.github/workflows/draft-release.yml

```diff
@@ -85,3 +88,31 @@ jobs:
         asset_path: ./tmp_CHANGELOG.md
         asset_name: CHANGELOG.md
         asset_content_type: text/markdown
+
+    - name: Upload ConjurSuite to release
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
```

Comment:
I think the other 3 files below this one also need:
- GITHUB_TOKEN
- upload_url